src/Tools/dotnet-monitor/GenerateApiKeyCommandHandler.cs

```diff
@@ -30,14 +30,14 @@ public Task<int> GenerateApiKey(CancellationToken token, IConsole console)
 
             byte[] hash = hashAlgorithm.ComputeHash(secret);
 
-            Console.Out.WriteLine(FormattableString.Invariant($"Authorization: {Monitoring.RestServer.AuthConstants.ApiKeySchema} {Convert.ToBase64String(secret)}"));
-            Console.Out.Write("HashedSecret ");
+            console.Out.WriteLine(FormattableString.Invariant($"Authorization: {Monitoring.RestServer.AuthConstants.ApiKeySchema} {Convert.ToBase64String(secret)}"));
+            console.Out.Write("ApiKeyHash: ");
```

Comment:
I'd proposed at some point to Wiktor that maybe we should just provide the object you need to add to the appsettings directly

```c#
keyHashString = BitConverter.ToString(hash).Replace("-","");
console.WriteLine("\"ApiAuthentication\": {");
console.WriteLine($"    \"ApiKeyHash\": \"{keyHashString}\",");
console.WriteLine("    \"ApiKeyHashType\": \"SHA256\"");
console.WriteLine("}");
```

Reply:
Our guidance has been to use key per file for these since they are likely to be mounted as secrets, so I'm not sure emitted a json object is ideal here.